packages/paste-website/src/components/homepage/HomeHeroIllustration.tsx

```diff
@@ -32,7 +33,7 @@ const IllustrationChildren: React.FC<{state: IllustrationStatesType}> = ({state}
  * - If prefers reduced motion, render static.
  */
 const HomeHeroIllustration: React.FC = () => {
-  const prefersReducedMotion = useReducedMotion();
+  const prefersReducedMotion = useReducedMotion() || inCypress();
```

Comment:
We use lottie and not our animation library, so disabling animations on the theme provider does nothing on the homepage

Reply:
suggestion: mind leaving the above comment inline in the code? Annoying suggestion I know, but it will make a future maintainer happy.

Reply:
done